tools/jetty/webapps/portal/index.html

```diff
@@ -1,75 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't understand .... this portal is currently in use? it is a simple re-direct with a 'nice' (ahem) logo to the /app URL?